docs/changelog/125060.yaml

```diff
@@ -0,0 +1,16 @@
+pr: 125060
+summary: Allow partial results by default in ES|QL
+area: ES|QL
+type: breaking
+issues: []
+
+breaking:
+  title: Allow partial results by default in ES|QL
+  area: ES|QL
+  details: >-
+    In earlier versions of {es}, ES|QL would fail the entire query if it encountered any error. ES|QL now returns partial results instead of failing when encountering errors.
+
+  impact: >-
+    Callers should check the `partial_results` flag returned in the response to determine if the result is partial or complete. If returning partial results is not desired, this option can be overridden per request via an `allow_partial_results` parameter in the query URL or globally via the cluster setting `esql.query.allow_partial_results`.
```

Comment:
The flag is named `is_partial` not `partial_results`.

Reply:
Good catch. I pushed https://github.com/elastic/elasticsearch/pull/125060/commits/cffeda5ac21bf5bcbde0aaddadad266c09a50cba